engine/classes/ElggSite.php

```diff
@@ -221,7 +214,7 @@ public function setDisplayName($displayName) {
 	 * Adds an entity to the site.
 	 *
 	 * This adds a 'member_of_site' relationship between between the entity and
-	 * the site. It does not change the site_guid of the entity.
+	 * the site.
```

Comment:
don't know if the relation was explicitly added for multisite purposes as timing of entity creation and relationship are different (or could be different)